core/src/test/scala/com/broadinstitute/dsp/Generators.scala

```diff
@@ -52,6 +52,12 @@ object Generators {
     googleProject <- genGoogleProject
   } yield KubernetesClusterToRemove(id, googleProject)
 
+  val genRuntimeWithWorkers: Gen[RuntimeWithWorkers] = for {
+    runtime <- genRuntime
+    num1 <- Gen.chooseNum(1, 100)
+    num2 <- Gen.chooseNum(1, 100)
```

Comment:
Does it make sense to include `0` in the range as it could help test some corner cases in the `spec`?

Reply:
we do not include 0 here on purpose, since this data will never occur due to database filters. Additionally, it adds some annoying conditionals, where the second number must be 0 if the first is.

Reply:
What do you mean by _database filters_?

Reply:
Did you mean the following line in the query

```
NOT rt.numberOfWorkers=0 AND rt.numberOfWorkers IS NOT NULL
```
?

Speaking of which, why are we filtering those out? Dataproc clusters can have only a master node and zero worker and secondary workers.